### PR TITLE
Refreshed shop theme

### PR DESCRIPTION
**Pull request by kulczy**
# Refreshed Shop Theme

The idea was to refresh the Shop Theme, much like Admin Theme, editing mostly css files, minimally touching twigs

# Home

shop-home

# Index

shop-index

# Show

shop-show

# Cart

shop-cart

# Login

shop-login

### CODE REVIEW

**Review by vvasiloi — APPROVED** [review]

**Comment by pjedrzejewski** [comment]
@kulczy Awesome, looks great to me! As great as it can be with Semantic UI. :D

**Review by Roshyo — APPROVED** [review]

File: src/Sylius/Bundle/ShopBundle/Resources/views/_footer.html.twig [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <footer id="footer" class="ui inverted vertical footer segment">
+    {% include "@SyliusShop/Layout/Footer/Grid/_pre_footer.html.twig" %}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/views/_footer.html.twig:2]: I know that the Symfony recommendation is to use `snake_case` for Twig templates, but in Sylius we have been using `camelCase` for ages:
```suggestion
    {% include "@SyliusShop/Layout/Footer/Grid/_preFooter.html.twig" %}
```

**Review by GSadee — APPROVED** [review]

File: src/Sylius/Bundle/ShopBundle/Resources/views/_footer.html.twig (hunk 1 above)
Comment (kulczy) [reply src/Sylius/Bundle/ShopBundle/Resources/views/_footer.html.twig:2]: yeah You're right, but in this particular folder, all files (don't know why) are named that way, so I didn't want to break the convention

Comment (GSadee) [reply src/Sylius/Bundle/ShopBundle/Resources/views/_footer.html.twig:2]: 😢

**Comment by GSadee** [comment]
Thank you, Szymon! :1st_place_medal:

**Pull request merged**